### DISCUSSION TITLE
Valheim dedicated server on a raspberry pi 3b+.

### DISCUSSION THREAD

**Discussion by Alteregoz** [GameServerManagers/LinuxGSM#3242] [General]
Hey, I'm currently looking into if I would be able to somehow run a dedicated server on a raspberry pi with the new 64bit beta release, any wiseheads in here able to humor me if there is a solution?
I have the pi setup with pi-hole and some retropie games, but it would be awesome to use it for even more.

**Comment by dgibbs64** [comment]
https://docs.linuxgsm.com/support/faq#will-linuxgsm-run-on-raspberry-pi
